test/e2e/storage/drivers/csi.go

```diff
@@ -168,18 +168,18 @@ func (h *hostpathCSIDriver) PrepareTest(f *framework.Framework) (*testsuites.Per
 
 // mockCSI
 type mockCSIDriver struct {
-	driverInfo     testsuites.DriverInfo
-	manifests      []string
-	podInfoVersion *string
-	attachable     bool
-	attachLimit    int
+	driverInfo  testsuites.DriverInfo
+	manifests   []string
+	podInfo     *string
```

Comment:
s/*string/*bool?

Reply:
I made it a string because this is the string argument that we pass to the cluster driver registrar container.  I first changed it to a bool and then realized I had to convert it back to a string... 

Let me know if you still prefer a bool

Reply:
I think a bool is better as that is consistent with what is defined in the API.  Otherwise it is confusing.

Reply:
done